fuel-tx/src/transaction/consensus_parameters/gas/default_gas_costs.rs

```diff
@@ -108,6 +107,10 @@ pub fn default_gas_costs() -> GasCostsValues {
             base: 15,
             units_per_gas: 103,
         },
+        croo: DependentCost::LightOperation {
+            base: 1,
+            units_per_gas: 1,
+        },
```

Comment:
Updating the costs is captured in a follow up task here: https://github.com/FuelLabs/fuel-core/issues/1660